components/ui/input-otp.tsx

```
@@ -64,4 +64,4 @@ function InputOTPSeparator({ ...props }: React.ComponentProps<"div">) {
   );
 }
 
-export { InputOTP, InputOTPGroup, InputOTPSlot, InputOTPSeparator };
+export { InputOTP, InputOTPGroup, InputOTPSlot,  };
```

ESLint

[error] Delete `,·`

(prettier/prettier)